styles.css

```diff
@@ -401,3 +397,230 @@ textarea {
   text-decoration: none;
   margin-bottom: 3rem;
 }
+
+@media only screen and (min-width: 768px) {
+  .container {
+    width: 90%;
+    padding: 1rem;
+    margin: 0 auto;
+    background: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
- [ ] Great job using flexbox to align the content of your webpage 👏 . Kindly make sure you also use `grid` to align the content of your webpage as stated in the project requirement 🙏 .This would also make your webpage responsive